Priyaa/HTML Basics Remake/Assignment-1.html

```diff
@@ -1,49 +1,60 @@
-        a reporter and assistant city editor at The Deseret News in Salt Lake City.[3]<br></p>
-        
-       <p><strike>Pusey</strike>Pusey worked for The Washington Post from 1928 to 1971, becoming associate editor in 1946, continuing to contribute occasional pieces until about two years before his death.
-        <br></p>
-        <p><b>Contents</b></p>
-        <p><a href="Books.html">1. Books</a><br>
-           <a href="References.html">2. References</a></p>
+   <p>Born and raised on a farm near Woodruff, Utah, Pusey was a Latter-day Saint. He attended the Latter-day Saints
+      University and graduated as a member of Phi Beta Kappa from the University of Utah after working on the college
+      newspaper. He later became
+      a reporter and assistant city editor at The Deseret News in Salt Lake City.[3]<br></p>
 
-           <h2>Books</h2>
-           <hr>
-           <ul>
-              <li>The Supreme Court Crisis(Macmillan, 1937)</li>
-              <li>Big Government: Can We Control It? (Harper, 1945)</li> 
-              <li>Charles Evans Hughes (2 vols., Macmillan, 1951)</li>
-               <li> Eisenhower, the President (Macmillan, 1956)</li></p>
-            </ul>
+   <p><strike>Pusey</strike>Pusey worked for The Washington Post from 1928 to 1971, becoming associate editor in 1946,
+      continuing to contribute occasional pieces until about two years before his death.
+      <br></p>
+   <p><b>Contents</b></p>
+   <p><a href="Books.html">1. Books</a><br>
+      <a href="References.html">2. References</a></p>
 
+   <h2>Books</h2>
+   <hr>
+   <ul>
+      <li>The Supreme Court Crisis(Macmillan, 1937)</li>
+      <li>Big Government: Can We Control It? (Harper, 1945)</li>
+      <li>Charles Evans Hughes (2 vols., Macmillan, 1951)</li>
+      <li> Eisenhower, the President (Macmillan, 1956)</li>
+      </p>
+   </ul>
 
-       <h2>References</h2>
-       <hr>
-       <ul>
-          <li>Smith, J. Y. (1985-11-23). "Merlo J. Pusey Dies at 83". The Washington Post. ISSN 0190-8286. Retrieved 2016-03-24.</li>
-          <li>Upi (1985-11-24). "Merlo J. Pusey Dies; Justice's Biographer Won a Pulitzer Prize". The New York Times. ISSN 0362-4331. Retrieved 2016-03-24.</li> 
-           <li>Latter-day Saint / Mormon Pulitzer Prize Winners". www.adherents.com. Retrieved 2016-03-24.</li>
-     </ul>
+
+   <h2>References</h2>
+   <hr>
+   <ul>
+      <li>Smith, J. Y. (1985-11-23). "Merlo J. Pusey Dies at 83". The Washington Post. ISSN 0190-8286. Retrieved
+         2016-03-24.</li>
+      <li>Upi (1985-11-24). "Merlo J. Pusey Dies; Justice's Biographer Won a Pulitzer Prize". The New York Times. ISSN
+         0362-4331. Retrieved 2016-03-24.</li>
+      <li>Latter-day Saint / Mormon Pulitzer Prize Winners". www.adherents.com. Retrieved 2016-03-24.</li>
+   </ul>
```

Comment:
Use `div` with `id` to link this section with `a` tag you used in the content section.